### ISSUE TITLE
Learning rate

### ISSUE THREAD

**Issue by mengjingyouling** [val-iisc/SDAT#6]
hi，Very solid work, I have a question:

Can the learning rate of discriminator be updated? Because in the function **class DomainDiscriminator(nn.Sequential):**   **"lr": 1.**

    def get_parameters(self) -> List[Dict]:
        return [{"params": self.parameters(), "lr": 1.}]

    **lr_scheduler_ad = LambdaLR(
        ad_optimizer, lambda x: args.lr * (1. + args.lr_gamma * float(x)) ** (-args.lr_decay))**

Does it work？

Because we found in the experiment that the discrimination loss remained stable when the epoch was very small, although the task loss was still decreasing.

We look forward to your reply
Thank you!

**Comment by mengjingyouling** [comment]
@abhinavagarwalla @jogendrakundu @rakmuga @rangwani-harsh

**Comment by rangwani-harsh** [comment]
Yes, the lr can be changed. However, we only experiment a little on the effect of lr on the discriminator. Also, we find that the loss should oscillate for adversarial learning to ensure good performance.

I just wanted to let you know that I'm closing this issue for now. Please re-open in case you have further queries.

Thanks
Harsh
